api/stripe-functions.js

```diff
@@ -67,9 +67,9 @@ function attachPlansToProducts(plans, products) {
  * @return {Array} An array of Stripe products that have 1+ plans
  * Each Stripe product contains an array of Stripe plans
  */
-function getProductsAndPlans() {
+async function getProductsAndPlans() {
   return Promise.all([
-    stripe.products.list({}), // Default returns 10 products, sorted by most recent creation date
+    stripe.products.list({  type:'service' }), // Default returns 10 products, sorted by most recent creation date
```

Comment:
Before without this change the array of product was empty, 
We need to add `type:service` in order to get our products.
[discussion here](https://stackoverflow.com/questions/64752137/node-stripe-products-list-return-empty)